examples/ipc_server_with_database.rs

```diff
@@ -36,7 +36,10 @@ type DbConn = Arc<Mutex<UnixStream>>;
 fn run_subprocess(cmd: &[&str]) -> std::process::Child {
     let exe_path = std::env::current_exe().unwrap();
 
-    let args: Vec<_> = ["run_main", "--", "--sub"].iter().chain(cmd.iter()).collect();
+    let args: Vec<_> = ["run_main", "--", "--sub"]
+        .iter()
+        .chain(cmd.iter())
+        .collect();
```

Comment:
So for example, this is fine and I don't really mind it, but then below it wants to put `.allow_running_tcp_servers()` back onto the same line. If this is easier to read here on separate lines why isn't it easier to read in that case?

I assume its heuristic is just based on line length but it doesn't really seem internally consistent.